packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -158,9 +158,13 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     }
 
     switch (imageSource) {
-      case SOURCE_CAMERA:
+      case SOURCE_CAMERA: {
+        NSInteger cameraDevice = [[_arguments objectForKey:@"cameraDevice"] intValue];
+        _device = (cameraDevice == 1) ? UIImagePickerControllerCameraDeviceFront
+                                      : UIImagePickerControllerCameraDeviceRear;
```

Comment:
This mapping from a magic number, as well as the hard-coded key, is now duplicated in two places. Please extract a small helper method that takes an arguments dictionary and returns the device, so that both locations can share the same code.

Reply:
done.